Packs/CarbonBlackDefense/Integrations/CarbonBlackDefenseV2/CarbonBlackDefenseV2.py

```diff
@@ -0,0 +1,1169 @@
+    blocked_hash = argToList(args.get('blocked_hash'))
+    device_external_ip = argToList(args.get('device_external_ip'))
+    device_id = argToList(args.get('device_id'))
+    device_internal_ip = argToList(args.get('device_internal_ip'))
+    device_name = argToList(args.get('device_name'))
+    device_os = argToList(args.get('device_os'))
+    event_type = argToList(args.get('event_type'))
+    parent_hash = argToList(args.get('parent_hash'))
+    parent_name = argToList(args.get('parent_name'))
+    parent_reputation = argToList(args.get('parent_reputation'))
+    process_cmdline = argToList(args.get('process_cmdline'))
+    process_guid = argToList(args.get('process_guid'))
+    process_hash = argToList(args.get('process_hash'))
+    process_name = argToList(args.get('process_name'))
+    process_pid = argToList(args.get('process_pid'))
+    process_reputation = argToList(args.get('process_reputation'))
+    process_start_time = argToList(args.get('process_start_time'))
+    process_terminated = argToList(args.get('process_terminated'))
+    process_username = argToList(args.get('process_username'))
+    sensor_action = argToList(args.get('sensor_action'))
+    query = args.get('query')
+    rows = arg_to_number(args.get('rows'))
+    start = arg_to_number(args.get('start'))
+    time_range = args.get('timerange')
+
+    if time_range:
+        time_range = json.loads(time_range)
+
+    res = client.get_events(alert_category, blocked_hash, device_external_ip, device_id, device_internal_ip,
+                            device_name, device_os, event_type, parent_hash, parent_name,
+                            parent_reputation, process_cmdline, process_guid, process_hash, process_name,
+                            process_pid, process_reputation, process_start_time, process_terminated,
+                            process_username, sensor_action, query, start, time_range, rows)
+
+    if type(res) is str:
+        return res
+
+    readable_output = tableToMarkdown('Carbon Black Defense Events Search',
+                                      res,
+                                      headerTransform=string_to_table_header)
+
+    return CommandResults(
+        outputs_prefix='CarbonBlackDefense.Events.Search',
+        outputs_key_field='job_id',
+        outputs=res,
+        readable_output=readable_output,
+        raw_response=res
+    )
+
+
+def find_events_results_command(client: Client, args: dict):
+    job_id = args.get('job_id')
+    rows = args.get('rows')
+    if not job_id:
+        return CommandResults(
+            readable_output="The job id can't be empty",
+            raw_response="The job id can't be empty"
+        )
+    if not rows:
+        rows = 10
+    res = client.get_events_results(job_id, rows)
+
+    headers = ['event_id', 'device_id', 'event_network_remote_port', 'event_network_remote_ipv4',
+               'event_network_local_ipv4', 'enriched_event_type']
+
+    human_readable = res.get('results')
```

Comment:
```suggestion
    human_readable = res.get('results', {})
```
Please also add a condition to check if there are no results, and return "No results returned" in such case.